ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -1450,7 +1460,7 @@ def test026_update_suborder_article(self):
     ### SYNTAX ERROR ###
 
     # will continue with us apply it from the second suborder & need test case number for it to apply from the second suborder
-    @parameterized.expand(['save_btn', 'cancel'])
+    #@parameterized.expand(['save_btn', 'cancel'])
```

Comment:
why this line is marked as a comment?

Reply:
it was a syntax error in the old order code but I handled it